### DISCUSSION TITLE
How can I enable the accuracyRing for LocationPuck when using a CustomLocationProvider?

### DISCUSSION THREAD

**Discussion by yuriydrobniy** [rnmapbox/maps#3670] [Q&A]
Hi everyone, Could you please give me some advice on how to enable "accuracyRing" for both platforms?

**Comment by Elter71** [comment]
Hi, the 'accuracyRing' is enabled by default. You can check it in the [UserLocationUpdates.tsx](https://github.com/rnmapbox/maps/blob/63d74fcb0fb7ad4e9e14d4859c8f57fc43dbe0dd/example/src/examples/UserLocation/UserLocationUpdates.tsx#L4) example; it works on both platforms.

**Reply by yuriydrobniy** [reply]
Thank you for your answer. But I forgot to mention that I use LocationPuck with CustomLocationProvider. How I can enable accuracyRing in this case?

**Reply by Elter71** [reply]
With LocationPuck, you can set up pulsing that will have the same radius as the accuracy value, but you might not see it (because it's so small). Try manually increasing the radius to see how it would look with larger accuracy values. 

Here is an [example](https://github.com/rnmapbox/maps/blob/main/example/src/examples/UserLocation/CustomNativeUserLocation.tsx)
